glide.yaml

```diff
@@ -2,8 +2,10 @@ package: github.com/kelseyhightower/confd
 import:
 - package: github.com/projectcalico/libcalico-go
   version: e3351395c934cee118999b9d29508e9280fa75ec
+# v3.3.3 + our fixes.
 - package: github.com/coreos/etcd
-  version: v3.3.0
+  version: e8287287893bd4080040e4fe68dae8a20476b095
```

Comment:
Do we have an upstream PR to fix this? I'd really rather not end up on a fork of etcd indefinitely. 

If so, probably worth a link to the PR and a comment saying we can switch back when it is merged.

Reply:
Trying my luck here: https://github.com/coreos/etcd/pull/9614

Upstream are wanting to do a bigger rework of that code, which is blocking their fix.